src/app/server/CommissioningWindowManager.h

```diff
@@ -191,6 +197,10 @@ class CommissioningWindowManager : public Messaging::UnsolicitedMessageHandler,
 
     bool mIsBLE = true;
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+    bool mJCM = false;
```

Comment:
Please document what this boolean means (that this is a commissioning window opened via the OpenJointCommissioningWindow command)?

Reply:
Added a comment in the latest commit.